tests/scripts/action.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
```

Comment:
how about naming the script `github-action-helper.sh`

Reply:
done.